specification/translation/data-plane/Azure.AI.DocumentTranslation/stable/2024-05-01/openapi.json

```diff
@@ -577,25 +572,96 @@
+            "required": true,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/DocumentTranslateContent.allowFallback"
+            "name": "category",
+            "in": "query",
+            "description": "A string specifying the category (domain) of the translation. This parameter is used to get translations \nfrom a customized system built with Custom Translator. Add the Category ID from your Custom Translator \nproject details to this parameter to use your deployed customized system. Default value is: general.",
+            "required": false,
+            "type": "string",
+            "default": "general"
           },
           {
-            "$ref": "#/parameters/Azure.Core.ClientRequestIdHeader"
+            "name": "allowFallback",
+            "in": "query",
+            "description": "Specifies that the service is allowed to fall back to a general system when a custom system doesn't exist. \nPossible values are: true (default) or false.",
+            "required": false,
+            "type": "boolean"
           },
           {
-            "$ref": "#/parameters/DocumentTranslateContent.document"
+            "name": "sourceLanguage",
+            "in": "formData",
+            "description": "Specifies source language of the input document.\nIf this parameter isn't specified, automatic language detection is applied to determine the source language.\nFor example if the source document is written in English, then use sourceLanguage=en",
+            "required": false,
+            "type": "string"
           },
           {
-            "$ref": "#/parameters/DocumentTranslateContent.glossary"
+            "name": "targetLanguage",
+            "in": "formData",
+            "description": "Specifies the language of the output document.\nThe target language must be one of the supported languages included in the translation scope.\nFor example if you want to translate the document in German language, then use targetLanguage=de ",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "category",
+            "in": "formData",
+            "description": "A string specifying the category (domain) of the translation. This parameter is used to get translations \nfrom a customized system built with Custom Translator. Add the Category ID from your Custom Translator \nproject details to this parameter to use your deployed customized system. Default value is: general.",
+            "required": false,
+            "type": "string",
+            "default": "general"
+          },
+          {
+            "name": "allowFallback",
+            "in": "formData",
+            "description": "Specifies that the service is allowed to fall back to a general system when a custom system doesn't exist. \nPossible values are: true (default) or false.",
+            "required": false,
+            "type": "boolean"
+          },
```

Comment:
@timotheeguerin is it a bug for multipart + @bodyRoot? it seems all query params are recognized as formdata part?